tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/TestingFrameworkTest.cs

```diff
@@ -0,0 +1,237 @@
+﻿// <copyright file="TestingFrameworkTest.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+#nullable enable
```

Comment:
Just FYI, we don't mandate `#nullable enable` in test code, as it tends to be more of a pain than it's worth. Not saying you _can't_, just FYI